packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -460,6 +465,32 @@ const OutputForm: React.FC<OutputFormProps> = ({
               />
             )}
 
+            {displayVersionModal && (
+              <ConfirmModal
+                title="Publish new version for the whole hub?"
+                cancelText="Cancel"
+                onCancel={() => setDiplayVersionModal(false)}
+                confirmText="Publish new version"
+                onSave={async () => {
+                  const skipPublishModal = true;
+                  const result = await save(skipPublishModal);
+                  if (!result) {
+                    setDiplayVersionModal(false);
+                  }
+                }}
+                description={
+                  <>
+                    Once published this output version will be available to all
```

Comment:
The copy is different from figma
<img width="1440" alt="Screenshot 2023-12-14 at 09 58 05" src="https://github.com/yldio/asap-hub/assets/16595804/b33dcba1-f71e-4582-a1db-9ed15b33e3de">